introduction_to_amazon_algorithms/jumpstart_semantic_segmentation/Amazon_JumpStart_Semantic_Segmentation_Extract_Image.ipynb

```diff
@@ -0,0 +1,575 @@
+{
```

Comment:
<p>Grammar things:</p><p>Lets -&gt; Let's</p><p>Original Pedestrian image with background  -&gt; original pedestrian image with the background included</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>